main.go

```diff
@@ -12,7 +13,13 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+var (
+	ignoreNormal = flag.Bool("ignore_normal", false, "ignore events of the normal type, they can be noisy")
```

Comment:
Dashes are more conventional IMO for command arguments. I'll also improve wording here:

```suggestion
	ignoreNormal = flag.Bool("ignore-normal", false, "ignore events of type 'Normal' to reduce noise")
```